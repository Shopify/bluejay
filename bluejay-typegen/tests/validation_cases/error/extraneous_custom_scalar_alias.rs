#[bluejay_typegen::typegen([
    type Query {
        foo: String
    }
])]
mod schema {
    type Foo = String;
}

fn main() {}
