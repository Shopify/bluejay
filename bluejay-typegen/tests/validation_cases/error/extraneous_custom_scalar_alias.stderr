error: No custom scalar definition named Foo
 --> tests/validation_cases/error/extraneous_custom_scalar_alias.rs:7:10
  |
7 |     type Foo = String;
  |          ^^^
